tests/dataloader/test_end_to_end_indexation_and_tokenization.py

```diff
@@ -0,0 +1,147 @@
+from collections import namedtuple
+from pathlib import Path
+
+import pytest
+
+from modalities.utils.verify_tokenization_constistency import (
```

Comment:
```suggestion
from modalities.utils.verify_tokenization_consistency import (
```
File needs to be renamed too ofc